feature_creation_scripts/transport_transects/flipOSNAP_renameArcticTransectsTag.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+  
+import shutil
+
+from geometric_features import GeometricFeatures, FeatureCollection
+from geometric_features.utils import write_feature_names_and_tags
+
+
+gf = GeometricFeatures(cacheLocation='../../geometric_data',
+                       remoteBranchOrTag='master')
+
+fcNew = FeatureCollection()
+
+# Flip OSNAP sections and rename their tag to 'arctic_transport_sections'
+fc = gf.read('ocean', 'transect', ['OSNAP section East'])
+properties = fc.features[0]['properties']
+properties['tags'] = 'arctic_transport_sections'
+geometry = fc.features[0]['geometry']
+coords = geometry['coordinates']
+geometry['coordinates'] = coords[::-1]
```

Comment:
I don't think this was successful.  Maybe try:
```suggestion
geometry['coordinates'] = [coords[0][::-1]]
```
I think the outer list only has one element, and the next level of lists is the one we want to reverse.  The same is true for "OSNAP section West".

Reply:
hmm, I actually looked at the new OSNAP transects and they were flipped when I did that.. weird.
Anyway, I'll try that.

Reply:
Okay, well just looking through the changes in the geojson files here, nothing serms to have been flipped.

Reply:
@xylar: the first version was actually correct, but I had accidentally run the flip script twice, and so it had flipped the sections again! I have now run it once more, gotten the correct sections and pushed again (just the OSNAP transect data).

Reply:
Okay, great!  I must have been thinking of a `MultiLineString`, rather than a `LineString`.